README.md

```diff
@@ -99,6 +106,43 @@ jobs:
 
 ```
 
+### Example: Pull-Request on another repo
+This example demonstrates how to create a pull-request in another repo. There are a few caveats such as the requirement of checking out the code with a [Github Personal Action Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token). There are some pretty advanced use cases for this such as building an app on every push to develop, updating the docker image tag and repo url in the config repo, and creating a pull-request to the config repo. After the pr in the config repo is merged a deployment is kicked off.
```

Comment:
Please correct it to "Personal Access Token" across this PR~

Reply:
done